wiki/Performance_points/Performance_points_commitee/en.md

```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Performance Points Committee
+
+The **Performance Points Committee** is a panel consisting of well-versed members of the development community whose main goal is to ensure that changes in the [performance points](/wiki/Performance_points) calculation algorithm could be facilitated, reviewed, and made effective in a fast and steady flow.
```

Comment:
```suggestion
The **Performance Points Committee** is a panel of well-versed players and developers who facilitate updates to the [performance points](/wiki/Performance_points) and [star rating](/wiki/Beatmapping/Star_rating) algorithms. Their goal is to help [osu! developers](/wiki/People/The_Team/Developers) push changes faster and with more community coordination.
```
just playing around with the order until it felt better; "facilitated" isn't usually by itself, "made effective" is sorta vague and implies more responsibility than they have i think
and split sentence between the "what" and the "why"
and link star rating as well

Reply:
"facilitate and approve updates"